### DISCUSSION TITLE
React Spectrum compatibility with React v19

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/react-spectrum#7510] [Q&A] [answered]
Has anybody tried using React Spectrum with React v19?  Any issues we should expect?

Thanks.

**Comment by snowystinger** [comment, marked as answer]
Any issues found will be tagged
https://github.com/adobe/react-spectrum/issues?q=is%3Aopen+is%3Aissue+label%3A%22react+19%22

our unit tests already run against React 19 and we build a storybook based on it as well
